pkg/commands/run.go

```diff
@@ -36,7 +36,8 @@ import (
 
 type RunCommand struct {
 	BaseCommand
-	cmd *instructions.RunCommand
+	cmd      *instructions.RunCommand
+	shdCache bool
```

Comment:
```suggestion
	cache bool
```

Or maybe `shouldCache`?

Reply:
`shdCache` is used for `COPY`command [here](https://github.com/GoogleContainerTools/kaniko/blob/main/pkg/commands/copy.go#L45), so I wanted to keep the same for consistency